jsonexample/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
 dev_dependencies:
   build_runner: ^1.0.0
   built_value_generator: ^6.1.5
-  json_serializable: ^1.0.0
+  json_serializable: ^2.2.1
```

Comment:
Whoa, yeah, thanks for catching this. I need to figure out how to hook into pub's "There's a new version you're not getting" warnings.

Reply:
Yeah, you could look at the output from `pub upgrade` – but flutter hides it TOO well.